src/app/_components/Links/socialMediaActions.js

```diff
@@ -0,0 +1,34 @@
+'use server';
+
+import { Facebook, Instagram } from '@icons';
+import { revalidateTag } from 'next/cache';
+import { NavigationUrl } from '@prisma/client';
+import { prisma } from '@/lib/db';
+
+export async function getSocialMediaList() {
+  const data = await prisma.navigation.findMany({
+    select: {
+      title: true,
+      href: true,
+    },
+  });
+
+  revalidateTag('/');
```

Comment:
Why do we revalidate the homepage every time we request social links?

Reply:
Let's just keep it as a temporary solution for now and will get back to it later, ok?